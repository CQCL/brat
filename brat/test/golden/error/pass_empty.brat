f(Nat) -> Nat
f = { (x => x), pass }
