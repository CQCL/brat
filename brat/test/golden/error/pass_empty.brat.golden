Error in test/golden/error/pass_empty.brat@2:17-19
f = { (x => x), .. }
                ^^

  Type error: pass is being given an empty row

